solutions/common-shared-flows/correlation-v1/sharedflowbundle/correlation-v1.xml

```diff
@@ -0,0 +1,19 @@
+<SharedFlowBundle revision="2" name="correlation-v1">
```

Comment:
I normally reduce this file down to the minimum that is required (only line 1, self-closing tag), because of how quick the contents becomes stale. e.g. `<Policies>` and line 1 is already referring to revision 2 which isn't relevant in this scenario.

Reply:
My comment above is actually with regards to the metadata file found in an **API Proxies**, but I assume the same applies to Shared Flows.